src/redux/apiFunctions.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable max-len */
+const missions = 'https://api.spacexdata.com/v3/missions';
+const rocketAPI = 'https://api.spacexdata.com/v3/rockets';
+const dragonData = 'https://api.spacexdata.com/v3/dragons';
+
+const loadRockets = async () => {
+  const response = await fetch(rocketAPI).then((res) => res.json()).then((GetRockets) => GetRockets);
+  const rocketData = response.map((rocket) => (
+    {
+      id: rocket.rocket_id,
+      name: rocket.rocket_name,
+      description: rocket.description,
+      image: rocket.flickr_images[0],
+    }
+  ));
+  return rocketData;
+};
+
+loadRockets();
```

Comment:
- [ ] [OPTIONAL] there is no need to call this method as it does nothing with the returned data.